compiler/ballerina-backend-jvm/src/main/ballerina/src/compiler_backend_jvm/jvm_instruction_gen.bal

```diff
@@ -17,6 +17,22 @@
 import ballerina/io;
 import ballerina/bir;
 import ballerina/jvm;
+import ballerina/runtime;
+
+boolean IS_BSTRING = runtime:getProperty("ballerina.bstring") != "";
+string BSTRING_VALUE = getBStingClassName();
```

Comment:
shall we use a ternary operator here instead of a method call? and can use the IS_BSTRING property right?